Util/ViolationFormatter.php

```diff
@@ -11,38 +11,11 @@
 
 namespace FOS\RestBundle\Util;
 
-use FOS\RestBundle\Controller\Annotations\Param;
-use FOS\RestBundle\Controller\Annotations\QueryParam;
-use Symfony\Component\Validator\ConstraintViolationInterface;
-use Symfony\Component\Validator\ConstraintViolationListInterface;
+use FOS\RestBundle\Validator\ViolationFormatter as BaseViolationFormatter;
 
-class ViolationFormatter implements ViolationFormatterInterface
+/**
+ * @deprecated since version 1.8, to be removed in 2.0. Use {@link BaseViolationFormatter} instead.
```

Comment:
since this is going into master the deprecation notices should go into the 1.7 branch